frontend/src/components/FormReview/constants.ts

```diff
@@ -0,0 +1,11 @@
+const inputText = {
```

Comment:
I think the variable can be renamed, `inputText` doesn't seems right for the purpose of it.